apps/api/src/app/events/usecases/process-subscriber/process-subscriber.usecase.ts

```diff
@@ -179,6 +193,23 @@ export class ProcessSubscriber {
 
     const preference = (await this.getSubscriberTemplatePreferenceUsecase.execute(buildCommand)).preference;
 
+    await this.createExecutionDetails.execute(
+      CreateExecutionDetailsCommand.create({
+        environmentId: environmentId,
+        organizationId: organizationId,
+        subscriberId: subscriberId,
+        notificationId: notificationId,
+        notificationTemplateId: template._id,
+        transactionId: transactionId,
+        detail: `Steps filtered by subscriber preferences`,
+        source: ExecutionDetailsSourceEnum.INTERNAL,
+        status: ExecutionDetailsStatusEnum.SUCCESS,
+        isTest: false,
+        isRetry: false,
+        raw: JSON.stringify(preference),
+      })
```

Comment:
We could create a mapper for cases like this one that the information doesn't come from a job.

Reply:
Yes we could but right now we only have one case for these and we do not yet know how next CreateExecutionDetailsCommand will be created and with what params... I suggest we wait with it until we know that 😄